### PR TITLE
Support for Django 1.8

### PR DESCRIPTION
**Pull request by izabeera**

### CODE REVIEW

File: splinter/driver/djangoclient.py [hunk 1]
```diff
@@ -8,7 +8,8 @@
 import os.path
 import re
 import sys
-import urlparse
+import six
```
Comment (andrewsmedina) [inline splinter/driver/djangoclient.py]: I believe that to use `six` from `django.utils.six` is a better way.

Comment (andrewsmedina) [inline splinter/driver/djangoclient.py]: It is not a problem. But it is needed to add the `six` dependencie in setup.py: https://github.com/kidosoft/splinter/blob/master/setup.py#L28

Comment (izabeera) [inline splinter/driver/djangoclient.py]: Thanks. Done :)

**Comment by andrewsmedina** [comment]
@izabeera thank you!

**Pull request merged**